prql-compiler/src/ast/expr.rs

```diff
@@ -64,11 +64,11 @@ pub struct Ident {
     pub name: String,
 }
 
-impl From<String> for Ident {
-    fn from(name: String) -> Self {
+impl Ident {
+    pub fn new_name<S: ToString>(name: S) -> Self {
```

Comment:
OK nice, thanks. Maybe I call this `from_name`?

Reply:
Now that I look at `new_name` I don't know what I was thinking when choosing it! `from_name` is much better.